src/Event/PartialEvent.php

```diff
@@ -94,6 +96,13 @@ public function addParticipation(BaseEventParticipation $participation)
         parent::addParticipation($participation);
     }
 
+    public function setParticipations(ArrayCollection $participations)
+    {
```

Comment:
And also, you should use the `Collection` type instead, instead of `ArrayCollection`